CONTRIBUTORS

```diff
@@ -5,6 +5,7 @@
 #
 Ace Nassri <anassri@google.com>
 Alexander Fenster <fenster@google.com>
+Alexander Fenster <github@fenster.name>
```

Comment:
I guess I'm not listed in .mailmap, hence the duplication.

Reply:
Feel free to update the `.mailmap` file throughout the `nodejs-*` repos and re-run `npm run generate-scaffolding` as desired.